app/src/main/java/com/example/kindergarten_management/helpers/SqlDatabaseHelper.java

```diff
@@ -1,148 +1,185 @@
-
+        db.execSQL("DROP TABLE IF EXISTS " + TABLE_STAFF_NAME);
+        db.execSQL("DROP TABLE IF EXISTS " + TABLE_KINDERGARTEN_NAME);
+        db.execSQL("DROP TABLE IF EXISTS " + TABLE_CLASS_NAME);
+        onCreate(db);
     }
 
-    /**
-     * Opens the database for writing.
-     */
     public void open() {
-        try {
-            db = getWritableDatabase();
-        } catch (Throwable t) {
-            t.printStackTrace();
-        }
+        db = getWritableDatabase();
     }
 
-    /**
-     * Closes the database.
-     */
     public void close() {
-        try {
+        if (db != null && db.isOpen()) {
             db.close();
-        } catch (Throwable t) {
-            t.printStackTrace();
         }
     }
 
-    public boolean deleteStaffMember(int staffMemberId) {
-        // todo - kfir
-        return false;
-    }
+    // CRUD operations for StaffMemberModel
 
     public boolean createStaffMember(StaffMemberModel staffMember) {
-        // todo - kfir
-        return false;
+        ContentValues values = new ContentValues();
+        values.put(STAFF_COLUMN_NAME, staffMember.getName());
+        values.put(STAFF_COLUMN_RULE, staffMember.getRule());
+        values.put(STAFF_COLUMN_START_WORKING_DATE, staffMember.getStartWorkingDate());
+        values.put(STAFF_COLUMN_ASSIGNED_KIND_ID, staffMember.getAssignedKindergartenId());
+        values.put(STAFF_COLUMN_ASSIGNED_CLASS_ID, staffMember.getAssignedClassId());
+
+        long result = db.insert(TABLE_STAFF_NAME, null, values);
+        return result != -1;
     }
 
     public StaffMemberModel readStaffMember(int staffMemberId) {
-        // todo - kfir
-        return null;
+        String query = "SELECT * FROM " + TABLE_STAFF_NAME + " WHERE " + STAFF_COLUMN_ID + " = ?";
+        Cursor cursor = db.rawQuery(query, new String[]{String.valueOf(staffMemberId)});
+        if (cursor != null && cursor.moveToFirst()) {
+            StaffMemberModel staffMember = new StaffMemberModel();
+            staffMember.setId(cursor.getInt(cursor.getColumnIndexOrThrow(STAFF_COLUMN_ID)));
+            staffMember.setName(cursor.getString(cursor.getColumnIndexOrThrow(STAFF_COLUMN_NAME)));
+            staffMember.setRule(cursor.getString(cursor.getColumnIndexOrThrow(STAFF_COLUMN_RULE)));
+            staffMember.setStartWorkingDate(cursor.getString(cursor.getColumnIndexOrThrow(STAFF_COLUMN_START_WORKING_DATE)));
+            staffMember.setAssignedKindergartenId(cursor.getInt(cursor.getColumnIndexOrThrow(STAFF_COLUMN_ASSIGNED_KIND_ID)));
+            staffMember.setAssignedClassId(cursor.getInt(cursor.getColumnIndexOrThrow(STAFF_COLUMN_ASSIGNED_CLASS_ID)));
+            cursor.close();
+            return staffMember;
+        } else {
+            return null;
+        }
     }
 
     public boolean updateStaffMember(StaffMemberModel staffMember) {
-        // todo - kfir
-        return false;
-    }
+        ContentValues values = new ContentValues();
```

Comment:
The code will brake, use the relevant constructor (that get the prameters)

**Same to all methods**